docs/v1/api.md

```diff
@@ -0,0 +1,239 @@
+# Service Catalog API
```

Comment:
I also dislike 'service controller API'.  A controller backs an API, it is not the API itself.